docs/pages/database-access/guides/cockroachdb-self-hosted.mdx

```diff
@@ -47,30 +48,42 @@ $ teleport db start \
   --uri=roach.example.com:26257 \
   --labels=env=dev
 ```
+</TabItem>
+<TabItem label="Teleport Cloud" scope={["cloud"]}>
+Start the Teleport Database Service, pointing the `--auth-server` flag at the address of your Teleport Cloud tenant, e.g., `mytenant.teleport.sh`.\
+
+```bash
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=mytenant.teleport.sh \
+  --name=roach \
+  --protocol=cockroachdb \
+  --uri=roach.example.com:26257 \
+  --labels=env=dev
+```
+</TabItem>
+</Tabs>
 
 <Admonition type="note">
   The `--auth-server` flag must point to the Teleport cluster's proxy endpoint
-  because database service always connects back to the cluster over a reverse
+  because the Database Service always connects back to the cluster over a reverse
   tunnel.
 </Admonition>
 
 <Admonition type="tip">
-  You can start Database service using configuration file instead of CLI flags.
+  You can start the Database Service using a configuration file instead of CLI flags.
   See [YAML reference](../reference/configuration.mdx).
 </Admonition>
 
-### Create Teleport user
+### Create a Teleport user
 
 (!docs/pages/includes/database-access/create-user.mdx!)
 
 ## Step 2/3. Configure CockroachDB
 
-### Create CockroachDB user
+### Create a CockroachDB user
 
-Teleport uses [mutual TLS authentication](https://www.cockroachlabs.com/docs/stable/authentication.html#client-authentication)
-with CockroachDB.
-
-Client certificate authentication is available to all CockroachDB users. If you
+Teleport uses mutual TLS authentication with CockroachDB. Client certificate authentication is available to all CockroachDB users. If you
```

Comment:
Any reason we don't want to keep the link to Cockroach docs inline? I see you added a footnote below but people aren't gonna see it until they read to the very end.

Reply:
I wanted to implement the style guide recommendation that, in guides, "Readers should be able to scroll down the guide and get to the end without leaving the page. Do not use an anchor or HTML link to describe some step - concisely, include the steps within the guide." (https://goteleport.com/docs/docs/#guides) Is there more information I could include in this paragraph while keeping the link in the Further Reading section? Otherwise, I'm happy to bring back the inline link if it makes the guide clearer.